### ISSUE TITLE
使用离线gpu版本启动报错

### ISSUE THREAD

**Issue by corey1202** [modelscope/FunASR#2112]
ubuntu系统启动离线GPU版本镜像,  registry.cn-hangzhou.aliyuncs.com/funasr_repo/funasr:funasr-runtime-sdk-gpu-0.1.1
使用FunASR/runtime/run_server.sh脚本,  指定 gpu
bash ./run_server.sh 时报错， 请问需要如何处理

[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.827 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.825, %4848, %4849, %4850), scope: __module.model.encoders/__module.model.encoders.40/__module.model.encoders.40.self_attn/__module.model.encoders.40.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.847 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.845, %4962, %4963, %4964), scope: __module.model.encoders/__module.model.encoders.41/__module.model.encoders.41.self_attn/__module.model.encoders.41.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.867 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.865, %5076, %5077, %5078), scope: __module.model.encoders/__module.model.encoders.42/__module.model.encoders.42.self_attn/__module.model.encoders.42.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

**Comment by lyblsgo** [comment]
[W shape_analysis.cpp:841] 这是warning，不是报错
